.buildkite/llvm-apple-pipeline.yml

```
@@ -1,3 +1,60 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_GVM_VERSION: "v0.5.1"
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
+  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  DOCKER_REGISTRY: "docker.elastic.co"
+  STAGING_IMAGE: "${DOCKER_REGISTRY}/observability-ci"
+  DOCKER_REGISTRY_SECRET_PATH: "secret/observability-team/ci/docker-registry/prod"
+  MAKEFILE: "go/llvm-apple"
+  FILES_TO_CHECK: ".buildkite/llvm-apple-pipeline.yml:.buildkite/scripts/llvm-apple:go/llvm-apple"
+  DOCKER_FILTER_REF: "*/*/golang-crossbuild:llvm-apple*"
+  BUILDX: "0"
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - label: ":linux: Build LLVM Apple / Ubuntu X86_64 - {{matrix.debianVersion}}"
+    key: "build-ubuntu-x86"
+    if: build.env("BUILDKITE_PULL_REQUEST") != "false" || build.source == "ui" || build.branch == "main" || build.branch =~ /[0-9]+\.[0-9]+/
```

Same as in the other step , should we add delimiters ? 
```suggestion
    if: build.env("BUILDKITE_PULL_REQUEST") != "false" || build.source == "ui" || build.branch == "main" || build.branch =~ /^[0-9]+\.[0-9]+$$/
```

---

There are others usages in the other pipelines, in case it is needed to be updated.